scene/main/scene_tree.h

```diff
@@ -461,6 +461,6 @@ class SceneTree : public MainLoop {
 
 VARIANT_ENUM_CAST(SceneTree::StretchMode);
 VARIANT_ENUM_CAST(SceneTree::StretchAspect);
-VARIANT_ENUM_CAST(SceneTree::CallGroupFlags);
+VARIANT_ENUM_CAST(SceneTree::GroupCallFlags);
```

Comment:
^ to avoid conflict with method `call_group_flags`. Also, I think this is the correct way to name it.